src/OpenTelemetry.AutoInstrumentation/Configurations/TracerInstrumentation.cs

```diff
@@ -131,4 +131,9 @@ internal enum TracerInstrumentation
     /// Elastic.Transport instrumentation.
     /// </summary>
     ElasticTransport = 19,
+
+    /// <summary>
+    /// Kafka client instrumentation
+    /// </summary>
+    Kafka = 20
```

Comment:
Extend SettingsTests.cs -> TracerSettings_Instrumentations_SupportedValues and Instrumentation.cs-> AddLazilyLoadedTraceInstrumentations

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3055/commits/eb633515a69b990c28e56b3f6d303ce4d640301d

Reply:
It seems that the test itself could rely on actual `enum`: so the test is always checking all values of the enum. Tracking via #3073